src/main/java/org/fedoraproject/javapackages/validator/checks/RpmFilesizeCheck.java

```diff
@@ -17,20 +17,20 @@ public RpmFilesizeCheck() {
     }
 
     @Override
-    public Collection<String> check(RpmFilesizeConfig config, RpmPathInfo rpm) throws IOException {
-        var result = new ArrayList<String>(0);
+    public CheckResult check(RpmFilesizeConfig config, RpmPathInfo rpm) throws IOException {
+        var result = new CheckResult();
 
         long filesize = Files.size(rpm.getPath());
-        String formattedFilesize = NumberFormat.getInstance(Locale.ENGLISH).format(filesize);
+        Decorated formattedFilesize = Decorated.actual(NumberFormat.getInstance(Locale.ENGLISH).format(filesize));
 
         if (!config.allowedFilesize(rpm, filesize)) {
-            result.add(failMessage("{0}: file size is: {1} bytes",
-                    textDecorate(rpm.getPath(), DECORATION_RPM),
-                    textDecorate(formattedFilesize, DECORATION_ACTUAL)));
+            result.add("{0}: file size is: {1} bytes",
+                    Decorated.rpm(rpm),
+                    Decorated.actual(formattedFilesize));
```

Comment:
There is double decoration taking place here and below, `formattedFilesize` is already decorated.